io.github.retgal.Dayon.yml

```diff
@@ -0,0 +1,50 @@
+app-id: io.github.retgal.Dayon
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: dayon_assisted.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --filesystem=home
```

Comment:
Does it actually need unrestricted home access?

Reply:
Yes, Dayon! allows files and clipboard content to be transfered between `assitant`and `assisted`.

Reply:
File access and transfer between `assistant` and `assisted` is rather limited in the flatpak version. Assistants will have to find some workarounds using the shell in order to help users running the `assisted` part as flatpak.